src/test/java/com/artipie/auth/AuthFromYamlTest.java

```diff
@@ -136,32 +140,46 @@ void doesNotAuthoriseByMainLogin() {
     /**
      * Composes yaml settings.
      * @param user User
+     * @param type Password type
      * @param pass Password
      * @return Settings
      */
-    private static YamlMapping settings(final String user, final String pass) {
+    private static YamlMapping settings(
+        final String user,
+        final String type,
+        final String pass) {
         return Yaml.createYamlMappingBuilder().add(
             "credentials",
             Yaml.createYamlMappingBuilder()
-                .add(user, Yaml.createYamlMappingBuilder().add("pass", pass).build())
-                .build()
+                .add(
+                    user,
+                    Yaml.createYamlMappingBuilder()
+                    .add("type", type)
+                    .add("pass", pass).build()
+                ).build()
         ).build();
     }
 
     /**
      * Composes yaml settings.
      * @param user User
+     * @param type Password type
      * @param pass Password
      * @param login Alternative login
      * @return Settings
+     * @checkstyle ParameterNumberCheck (3 lines)
      */
-    private static YamlMapping settings(final String user, final String pass, final String login) {
+    private static YamlMapping settings(final String user, final String type, final String pass,
+        final String login) {
         return Yaml.createYamlMappingBuilder().add(
             "credentials",
             Yaml.createYamlMappingBuilder()
                 .add(
                     user,
-                    Yaml.createYamlMappingBuilder().add("pass", pass).add("login", login).build()
+                    Yaml.createYamlMappingBuilder()
+                    .add("type", type)
+                    .add("pass", pass)
+                    .add("login", login).build()
```

Comment:
@paulodamaso where is `login` used? I believe its redundant

Reply:
@Vatavuk `login` will be used to provide an alternative login (there is a puzzle in `AuthFromYaml`)